Maddori.Apple/Maddori.Apple/Screen/Home/Home/TeamManageViewController.swift

```diff
@@ -16,14 +16,22 @@ final class TeamManageViewController: BaseViewController {
         static let teamSectionHeight = 59
         static let teamSectionSpacing = 8
         static let teamSectionPadding = 28
-        static let teamTitleLabelHeight = 50
+        static let teamTitleLableHeight = 50
```

Comment:
p1;
```suggestion
        static let teamTitleLabelHeight = 50
```
기존 코드가 맞는 것 같아용!